### PR TITLE
TST: Add test to check for legacy plan names

### PR DESCRIPTION
**Pull request by stuartcampbell**
Add a test to check for that legacy plan names exist.

### CODE REVIEW

File: bluesky/tests/test_legacy_plans.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import bluesky.plans as bp
+
+def test_legacy_plan_names():
+    assert bp.outer_product_scan is bp.grid_scan
+    assert bp.relative_outer_product_scan is bp.rel_grid_scan
+    assert bp.relative_scan is bp.rel_scan
+    assert bp.relative_spiral is bp.rel_spiral
+    assert bp.relative_spiral_fermat is bp.rel_spiral_fermat
+    assert bp.relative_list_scan is bp.rel_list_scan
+    assert relative_log_scan is rel_log_scan
```
Comment (danielballan) [inline bluesky/tests/test_legacy_plans.py]: Tests fail because `bp.` is missing here and on the line below.

Comment (stuartcampbell) [reply bluesky/tests/test_legacy_plans.py]: Doh, sorry.  That's what you get for rushing on the way to a meeting with the facility director! :)

Comment (danielballan) [reply bluesky/tests/test_legacy_plans.py]: :- D

**Comment by danielballan** [comment]
Thanks!

**Comment by danielballan** [comment]
Thanks!

**Pull request merged**